### ISSUE TITLE
Omg files decrypt

### ISSUE THREAD

**Issue by fshuetli** [linux-minidisc/linux-minidisc#76]
Back in the day I recorded loads of concerts using a Sony MiniDisc. I then used the software sonicstage to copy the openmg atrac audio files *.omg from the minidisc to my PC to archive them on a hard disk.
What I did not know: The files were protected with DRM and the key was stored in a file which I did not backup.
Since years I am looking for a way to unlock/crack those files (which are my own recordings of my music).
I would be so thankful and also ready to pay, if anyone knows a solution or a pointer to a solution.

**Comment by thp** [comment]
I guess you don't have the original MiniDisc media anymore? A MZ-RH1 should be able to upload files from a disc DRM-free. Not sure how feasible it is to brute force the OMG key.

**Comment by fshuetli** [comment]
Correct, I don't have the original media anymore. Just the *.OMG files from Sonicstage. This is why probably I would need a leaked master key or a brute forced key probably...
